test/initex_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright (c) 2018-2021 the Go Library for Virtual Disk Development Kit contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"github.com/vmware/virtual-disks/pkg/disklib"
+	"os"
+	"testing"
+)
+
+// To run this test case, the following parameters are required:
+// LIBPATH: vddk library path
+// CONFIGFILE: path for config file which contains customized log level setting, eg: verbosevixDiskLib.transport.LogLevel=4
+// VC IP, THUMBPRINT, USERNAME, PASSWORD, FCDID, DATASTORE, IDENTITY
```

Comment:
What does `IDENTITY` means and what are we supposed to provide to this parameter? Is it required? BTW, is `DATASTORE` optional? Would you please clearly document here whether each parameter above is required or optional?

Reply:
Add comment for IDENTITY parameter. All the parameters listed above are required in this testcase.